app/styles/custom-utilities/_text.scss

```diff
@@ -37,3 +37,8 @@
 .vlc-u-font-weight-bold {
   font-weight: bold;
 }
+
+.vlc-u-align-right {
+  text-align: right !important;
+}
+
```

Comment:
Kunnen we hier geen AUK css voor gebruiken.. ik zou verwachten dat zulke alignment utils er al inzitten.. indien niet best toevoegen in die css package en die klasse daaruit gebruiken.. zo blijven we niet met .vlc utils werken.